loopin.go

```diff
@@ -681,6 +691,16 @@ func (s *loopInSwap) publishOnChainHtlc(ctx context.Context) (bool, error) {
 
 }
 
+// getTxFee calculates our fee for a transaction that we have broadcast. We use
+// sat per kvbyte because this is what lnd uses, and we will run into rounding
+// issues if we do not use the same fee rate as lnd.
+func getTxFee(tx *wire.MsgTx, fee chainfee.SatPerKVByte) btcutil.Amount {
+	btcTx := btcutil.NewTx(tx)
+	vsize := mempool.GetTxVirtualSize(btcTx)
```

Comment:
I know we can't get this any more accurately if we can't look at the spent UTXOs (since we don't have a chain backend to query). This might be slightly off from the actual fee though since the fee rate is not an exact outcome in the coin selection. For our use case that might be okay though.

Reply:
Would be cool if we could start using the PSBT interfaces in the future so we could actually look at the UTXO information for stuff like this.